src/pyOpenMS/pxds/MetaboTargetedAssay.pxd

```diff
@@ -9,8 +9,8 @@ cdef extern from "<OpenMS/ANALYSIS/TARGETED/MetaboTargetedAssay.h>" namespace "O
 
     cdef cppclass MetaboTargetedAssay "OpenMS::MetaboTargetedAssay":
 
-       MetaboTargetedAssay() nogil except +
-       MetaboTargetedAssay(MetaboTargetedAssay &) nogil except + # compiler
+       MetaboTargetedAssay() nogil except + # wrap-doc:This class provides methods for the extraction of targeted assays for metabolomics
+       MetaboTargetedAssay(MetaboTargetedAssay) nogil except + # wrap-ignore
```

Comment:
Did something go wrong with the merging? 
Should be:
`MetaboTargetedAssay(MetaboTargetedAssay &) nogil except + # compiler`

Reply:
See previous copy ctor PR
https://github.com/OpenMS/OpenMS/pull/5460/files?file-filters%5B%5D=.pxd

Reply:
Ohh sorry. I was not sure about the comment `# compiler`. I will revert.

Reply:
The most important part is:
MetaboTargetedAssay(MetaboTargetedAssay **&**) nogil except + # compiler

We changed the copy constructors of the pyopenms bindings to the correct format. Basically taking in a reference instead of a copy. 

The additional statements mean:
# compiler: This means that there was no user-specified constructor/copy constructor - automatically added by the compiler.
# private/deleted: Not implemented
# protected: Not available in pyopenms

You can also have a look at the description here:
https://github.com/OpenMS/OpenMS/pull/5460

You can use the PR also as a reference point how the copy constructor should look like.